examples/rapids-image/README.md

```diff
@@ -0,0 +1,81 @@
+## RAPIDS.ai Image
+
+### Overview
+
+A RAPIDS kernel that is a compatible as a Bring-Your-Own Image in SageMaker Studio.
+
+The image is based on the [RAPIDS image](https://hub.docker.com/r/rapidsai/rapidsai/).
+
+### Building the image
+
+Build the Docker image and push to Amazon ECR. 
+```
+# Modify these as required. The Docker registry endpoint can be tuned based on your current region from https://docs.aws.amazon.com/general/latest/gr/ecr.html#ecr-docker-endpoints
+REGION=<aws-region>
+ACCOUNT_ID=<account-id>
+
+
+# Build the image
+IMAGE_NAME=custom-rapids
+aws --region ${REGION} ecr get-login-password | docker login --username AWS --password-stdin ${ACCOUNT_ID}.dkr.ecr.${REGION}.amazonaws.com/smstudio-custom
+docker build . -t ${IMAGE_NAME} -t ${ACCOUNT_ID}.dkr.ecr.${REGION}.amazonaws.com/smstudio-custom:${IMAGE_NAME}
+```
+
+```
+docker push ${ACCOUNT_ID}.dkr.ecr.${REGION}.amazonaws.com/smstudio-custom:${IMAGE_NAME}
+```
+
+### Using with SageMaker Studio
+
+```
+pip install sagemaker-studio-image-build
+```
+
+then build and push using sm-docker build (make sure you are in this directory with the dockerfile)
+
+```
+sm-docker build .
+```
+
+
+Create a SageMaker Image with the image in ECR. 
+
+```
+# Role in your account to be used for the SageMaker Image
+ROLE_ARN=<role-arn>
+
+aws --region ${REGION} sagemaker create-image \
+    --image-name ${IMAGE_NAME} \
+    --role-arn ${ROLE_ARN}
+
+aws --region ${REGION} sagemaker create-image-version \
+    --image-name ${IMAGE_NAME} \
+    --base-image "${ACCOUNT_ID}.dkr.ecr.${REGION}.amazonaws.com/smstudio-custom:${IMAGE_NAME}"
+
+# Verify the image-version is created successfully. Do NOT proceed if image-version is in CREATE_FAILED state or in any other state apart from CREATED.
+aws --region ${REGION} sagemaker describe-image-version --image-name ${IMAGE_NAME}
+```
+
+Create a AppImageConfig for this image
+
+```
+aws --region ${REGION} sagemaker create-app-image-config --cli-input-json file://app-image-config-input.json
+
+```
+
+Create a Domain, providing the SageMaker Image and AppImageConfig in the Domain creation. Replace the placeholders for VPC ID, Subnet IDs, and Execution Role in `create-domain-input.json`
+
+```
+aws --region ${REGION} sagemaker create-domain --cli-input-json file://create-domain-input.json
+```
+
+If you have an existing Domain, you can also use the `update-domain`
+
+```
+aws --region ${REGION} sagemaker update-domain --cli-input-json file://update-domain-input.json
+```
+
+### Note
+
+- Please use the included `Python [conda env:rapids]' kernel within Studio
```

Comment:
I'm confused by this - In the `AppImageConfig` we say that the customer should use the `python3` kernel and here we say that the kernel from the Conda environment should be used.

The kernel name that the customer will use should be the one that goes in `AppImageConfig`.

Reply:
fixed everything in [this commit](https://github.com/aws-samples/sagemaker-studio-custom-image-samples/pull/2/commits/51ba07eb4f75d88842159975b3b8b3391d848f01) except this one --- What should be the value for name in KernelSpecs inteh AppImageConfig if users need to select “Python [conda env:rapids]” ?

Reply:
Guessing it is "python" based on what should be picked, since the other examples just say "Python3" even though there are many python3 kernels available to choose from. Related to [this commit](https://github.com/aws-samples/sagemaker-studio-custom-image-samples/pull/2/commits/a797a6c688ec786f8b1e823d07be09d4a75a0898)

Reply:
changed kernelspec name to conda-env-rapids-py